### PR TITLE
add documentation folder and exemplify custom abi

### PR DESCRIPTION
**Pull request by moshmage**
[dev-283](https://taikai.atlassian.net/jira/software/c/projects/DEV/boards/45?modal=detail&selectedIssue=DEV-283)

### CODE REVIEW

**Comment by hvasconcelos** [comment]
Create links from the main README to the custom-abi.md .

File: docs/custom-abi.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Using @taikai/dappkit with a custom ABI
```
Comment (hvasconcelos) [inline docs/custom-abi.md]: with a custom smart contract ????

File: docs/custom-abi.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+AliceCustomModel.web3Connection.start();
+AliceCustomModel.loadAbi();
+
+const tx = await AliceCustomModel.deploy({data: AliceCustomContract.bytecode, arguments: []}, AliceCustomModel.web3Connection.Account);
+
+console.log('Contract deployed; Address: ', tx.contractAddress);
+```
+
+### Using a custom contrat
```
Comment (hvasconcelos) [inline docs/custom-abi.md]: contrat typo

**Comment by hvasconcelos** [comment]
Would not be better to have separate instructions to deploy a custom smart contract  from the instructions to interact with a custom smart contract ?

**Comment by moshmage** [comment]
> Would not be better to have separate instructions to deploy a custom smart contract from the instructions to interact with a custom smart contract ?

I've added a index of sorts to the custom-abi file, this should suffice to clear the doubts -- having two different files would leave both of them too empty or too repetitive -- but we can add more spacing between them if needed. Please recheck :)

In the meanwhile, I've added the "how to connect" as well as it was small enough

**Pull request merged**